packages/cli/src/bin/ceramic.ts

```diff
@@ -1,15 +1,15 @@
 import program from 'commander'
 
-import { CeramicCliUtils, DEFAULT_PINNING_STORE_PATH } from "../ceramic-cli-utils"
+import { CeramicCliUtils } from "../ceramic-cli-utils"
 
 program
     .command('daemon')
     .option('--ipfs-api <url>', 'The ipfs http api to use')
     .option('--ethereum-rpc <url>', 'The Ethereum RPC URL used for communicating with Ethereum blockchain')
     .option('--anchor-service-api <url>', 'The anchor service URL to use')
     .option('--validate-docs', 'Validate documents according to their schemas. It is enabled by default')
-    .option('--pinning <url...>', 'Pinning endpoints')
-    .option('--pinning-store-path <url>', `The directory path used for pinning service. Defaults to WORKING_DIR/${DEFAULT_PINNING_STORE_PATH}`)
+    .option('--pinning-endpoints <url...>', 'Pinning endpoints')
```

Comment:
I think this should be a singular since you have to pass the flag multiple times to specify multiple endpoints.